targets/targets.json

```diff
@@ -7582,25 +7583,21 @@
             "STDIO_MESSAGES",
             "LPTICKER",
             "SLEEP",
-            "FLASH",
-            "MPU"
+            "FLASH"
         ],
-        "overrides": {
-            "mpu-rom-end": "0x1fffffff"
-        },
         "release_versions": ["5"],
         "extra_labels": ["Cypress", "PSOC6"],
         "public": false
     },
     "MCU_PSOC6_M0": {
         "inherits": ["MCU_PSOC6"],
         "core": "Cortex-M0+",
-        "macros": ["MCU_PSOC6_M0"],
+        "macros_add": ["MCU_PSOC6_M0"],
```

Comment:
i think this one is not related to our MPU configuration, but clearly a necessary change!
Can you consider splitting it to separate commit?